gtfs-realtime/CHANGES.md

```diff
@@ -0,0 +1,80 @@
+The GTFS-realtime specification is not set in stone. Instead, it is an open specification developed and maintained by the community of transit agencies, developers, and other stakeholders who use GTFS-realtime. It is expected that this community of producers and consumers of GTFS-realtime data will have proposals for extending the spec to enable new capabilities. To help manage that process, the following procedures and guidelines have been established.
+
+### Specification amendment process
+The official specification, reference and documentation are written in English. If a translation to a different language differs from the English original, the latter takes precedence. All communication is performed in English.
+
+1. Create a git branch with update of all relevant parts of protocol definition, specification and documentation files (except for translations).
+1. Create pull request on https://github.com/google/transit. Pull request must contain an extended description of the patch. The creator of the pull request becomes the _advocate_.
+1. Once pull request is registered, it must be announced by its advocate in the [GTFS Realtime mailing list](https://groups.google.com/forum/#!forum/gtfs-realtime). Once announced, the pull request is considered a proposal.
+  - Advocate is a contributor, thus they should sign [CLA](../CONTRIBUTING.md) before pull request can be accepted.
+1. The discussion of the proposal follows. Pull request comments should be used as the sole discussion forum.
+  - The discussion lasts for as long as the advocate considers required but at least 7 calendar days.
+  - The advocate is responsible for timely update of the proposal based on the comments for which they agree to.
+  - At any point in time the advocate can claim proposal abandoned.
+1. The advocate can call for a vote on a version of the proposal at any point in time but no earlier than 7 calendar days, and no later than 30 calendar days after the proposal was announced.
```

Comment:
s/time but no earlier than 7 calendar days/time following the initial 7-day interval required for discussion/